.github/publish_branch.yaml

```diff
@@ -0,0 +1,24 @@
+name: Manual Publish
```

Comment:
this file is not in the workflows directory

Reply:
Oops .. fixing it